components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/DynamicX509TrustManager.java

```diff
@@ -0,0 +1,141 @@
+/*
+*  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Correct the copyright header with correct format, refer to one of the recently created component to get the correct format.